doc/rfc/1-index.txt

```diff
@@ -17,201 +17,3 @@ draft-wilson-wccp-v2-12-oct-2001.txt
 
 draft-vinod-carp-v1-03.txt
 	Microsoft CARP peering algorithm
-
-proxy-protocol.txt
-	The PROXY protocol, Versions 1 & 2
```

Comment:
Please delete the stale copy of the PROXY protocol specs as well. If I am reading the current GitHub PR rendering correctly, the `doc/rfc/proxy-protocol.txt` file has not been removed.

Reply:
This I kept because that document is still somewhat dynamic with each Haproxy release and the file we have is the one Squid was implemented from. Same goes for the IETF Draft documents retained.

Reply:
> the file we have is the one Squid was implemented from

I am pretty sure the above assertion is false or misleading. While earlier PROXY protocol implementation in Squid may have been based on that specific revision of the specs, significant bug fixes and improvements that followed used (and should continue to use) then-current (i.e. contemporary) revisions.

Moreover, it does not really matter much what document revision existed at the time of a given Squid commit[^1]. What really matters is whether Squid interoperates and, to a lesser extent, follows the latest revision of the specs. Keeping any revision of the specs in the Squid repository does not help with any of that. And keeping a stale revision can only make things worse.

[^1]: If that information is really needed, it can usually be recovered without keeping a copy of the specs in the Squid repository.